metricbeat/module/consul/agent/agent_test.go

```diff
@@ -0,0 +1,4043 @@
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+                "service": "resolve-epmd"
+            }
+        },
+        {
+            "Name": "consul.consul.catalog.service.query",
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+                "service": "resolve-lb"
+            }
+        },
+        {
+            "Name": "consul.consul.catalog.service.query",
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+                "service": "rider"
+            }
+        },
+        {
+            "Name": "consul.consul.catalog.service.query",
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+                "service": "rider-beam"
+            }
+        },
+        {
+            "Name": "consul.consul.catalog.service.query",
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+                "service": "rider-cp-beam"
+            }
+        },
+        {
+            "Name": "consul.consul.catalog.service.query",
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+                "service": "rider-cp-epmd"
+            }
+        },
+        {
+            "Name": "consul.consul.catalog.service.query",
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+                "service": "rider-critical-path"
+            }
+        },
+        {
+            "Name": "consul.consul.catalog.service.query",
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+                "service": "rider-critical-path-lb"
+            }
+        },
+        {
+            "Name": "consul.consul.catalog.service.query",
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+                "service": "rider-epmd"
+            }
+        },
+        {
+            "Name": "consul.consul.catalog.service.query",
+            "Count": 1,
+            "Sum": 1,
+            "Min": 1,
+            "Max": 1,
+            "Mean": 1,
+            "Stddev": 0,
+            "Labels": {
+      
```

Comment:
Sorry, this is just a personal convention that I'm struggling to get rid of. Just to clarify, I started to do it because I clearly wanted to see visually which methods I was testing in the source code and in the console. I'll try to stop doing it, it's just it has been too many years doing it :sweat_smile: